### ISSUE TITLE
复现问题

### ISSUE THREAD

**Issue by qqa111** [YvanYin/DrivingWorld#3]
您好，基于您开源的代码，复现过程中存在一些问题，希望您有时间帮忙解答一下：
1、数据准备代码：我认为 datasets/dataset_nuplan.py  以下代码中 语句 poses_new.append(.......)    是否应该如第二张图修改
2、请问一下 datasets/dataset_utils.py 文件中函数 get_meta_data(poses) 中对poses的处理是什么意思呢，返回值 rel_poises , rel_yaws是什么意思，是车辆在哪个坐标系的坐标吗？
3、图像预处理时将其归一化至（-1，1），是有什么特殊意义吗？
![2](https://github.com/user-attachments/assets/d9d4521f-1b08-4f6d-b032-6eba5b8d63a5)
![1](https://github.com/user-attachments/assets/532f672c-e1d2-4285-8e60-6b7c23b5f3f7)
期待您的回复。

**Comment by qqa111** [comment]
还想请问一下训练代码最近有开源的打算吗

**Comment by huxiaotaostasy** [comment]
Hi, thanks for your question.

1. Yes, thanks for your correction, we have updated the code.

2. rel_poses , rel_yaws, **represent the pose and yaw relative to the previous frame.** For this part, you can refer to Vehicle Pose Tokenizer in 3.1 of the paper. 
_'Instead of global poses, we adopt the relative poses between adjacent time steps. This is because global poses present a significant challenge due to the increasing magnitude of absolute pose values ​​over long-term sequences. This growth makes normalization difficult and reduces model robustness. As sequences grow longer, managing these large pose values ​​becomes increasingly difficult, hindering effective long-term video generation.'_

3. We refer to many popular encoder codes, which will normalize the image to (-1, 1). 
    a. Normalization can make the gradient update smoother during training, thereby accelerating the convergence of the model. The smaller the data distribution, the less large fluctuations the model will experience during learning. 
    b. Many deep learning models are sensitive to the range of input data. Normalizing the image pixel values ​​to a uniform range can help the model learn features better. 
    c. When doing floating point calculations, a large range of values ​​can cause overflow or underflow. Normalization helps maintain numerical stability. 
    d. Different datasets may have different pixel value ranges. Normalizing all images to the same range helps maintain consistency, making the data processing and model training process more uniform across different images.

4. We are still organizing the training code, and as you know, organizing the code usually requires a lot of work. Once it is organized, we will open source it.

Thank you again for your attention to our work. I hope my answer is helpful to you. If you have any questions, please feel free to contact us.
